source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -204,463 +204,166 @@ Persistent Volumes
    For more information on AKS disk types, see :azure-docs:`Azure disk types <virtual-machines/disk-types>`.
 
 
-Deploy a Tenant using the MinIO Operator Console
-------------------------------------------------
+Deploy a MinIO Tenant using Kustomize
+-------------------------------------
 
-To deploy a tenant from the MinIO Operator Console, complete the following steps in order:
+The following procedure uses ``kubectl -k`` to deploy a MinIO Tenant using the ``base`` Kustomization template in the MinIO Operator Github repository.
 
-:ref:`create-tenant-access-minio-operator-console`
+You can select a different base or pre-built template from the repository as your starting point, or build your own Kustomization resources using the MinIO Custom Resource Documentation.
```

Comment:
Is there a good link for the DIY option?

Reply:
Yeah, will do that in this pass